src/Common/ItechArt.Common/Enums/LogLevel.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ItechArt.Common;
+
+public enum LogLevel
```

Comment:
reorganize folder structure to contain everything related to logging in single folder (and its subfolders)